Remotion/ObjectBinding/Web/UI/Controls/BocListImplementation/Rendering/BocColumnRendererBase.cs

```diff
@@ -388,13 +388,16 @@ private void RenderDataCell (BocColumnRenderingContext<TBocColumnDefinition> ren
           tooltipStringBuilder.AppendLine(validationFailure.Failure.ErrorMessage);
         }
 
-        renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Id, BocRowRenderer.GetCellIDForValidationMarker(renderingContext.Control, arguments.RowIndex, renderingContext.VisibleColumnIndex));
         renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Title, tooltipStringBuilder.ToString());
-        renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Tabindex, "-1");
         renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Class, CssClasses.ValidationErrorMarker);
         renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute2.AriaHidden, HtmlAriaHiddenAttributeValue.True);
         renderingContext.Writer.RenderBeginTag(HtmlTextWriterTag.Span);
 
+        renderingContext.Writer.AddAttribute(HtmlTextWriterAttribute.Id, BocRowRenderer.GetCellIDForValidationMarker(renderingContext.Control, arguments.RowIndex, renderingContext.VisibleColumnIndex));
```

Comment:
since we have a special intent here, making a zero-size span, please add a code comment to explain the idea and technique for future devs seeing the code.